fetch.bs

```diff
@@ -813,23 +802,28 @@ the empty string,
 "<code>worker</code>", or
 "<code>xslt</code>". Unless stated otherwise it is the empty string.
 
+<p>A <dfn for=export>script-like destination</dfn> is "<code>script</code>",
+"<code>serviceworker</code>", "<code>sharedworker</code>", or "<code>worker</code>".
```

Comment:
If we made "`xslt`" part of this, then [nosniff](https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-nosniff?) would fail.

And adding "`xslt`" to [MIME type blocking](https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-mime-type?) probably doesn't help either as XSLT already requires a correct MIME type?